src/components/Login.js

```diff
@@ -126,10 +126,10 @@ export const Login = Form.create({ name: "normal_register" })(LoginForm);
 
 const StyledContainer = styled.div`
   display: flex;
-  justify-content: center;
+  flex-direction: column;
   align-items: center;
-  min-height: 80vh;
```

Comment:
looks like this may have been removed by accident

Reply:
We updated that together, not sure how it got lost.  updated